### ISSUE TITLE
Workspaces export...  bug of 2019..........................................

### ISSUE THREAD

**Issue by Vins1417** [iNavFlight/blackbox-log-viewer#95]
Unable to export the workspace, the button exists, but all it does is write null... are there any developers still working on this software? there is an issue from 2019 on this subject ?... how does UAV tech provide us with these files on its Patreon if we can't do it ??

**Comment by Tonynux** [comment]
Still export not working!

**Comment by druckgott** [comment]
The Function for CustomExport is not Availibe.

If you set the Export with default value in js/main.js then you get an export, but it´s not the current version, it´s a default example.

![grafik](https://github.com/iNavFlight/blackbox-log-viewer/assets/18383738/ab8149f8-e4cb-4e75-99f3-c94a5801b5cf)

If you search for workspaceGraphConfigs you will not find a function. so always item in this area is emptyp 
https://github.com/iNavFlight/blackbox-log-viewer/blob/721ad97f21e9bbf62cdec8031ebc76be379293a3/js/main.js#L796
You will always get this output:
https://github.com/iNavFlight/blackbox-log-viewer/blob/721ad97f21e9bbf62cdec8031ebc76be379293a3/js/main.js#L800

If you search for graphConfig you will finde a function
https://github.com/iNavFlight/blackbox-log-viewer/blob/721ad97f21e9bbf62cdec8031ebc76be379293a3/js/graph_config.js#L4

So anybody have to write the custom function to export this.

**Comment by druckgott** [comment]
Fixed it:
https://github.com/iNavFlight/blackbox-log-viewer/pull/99

**Comment by Tonynux** [comment]
Thank you.

**Comment by druckgott** [comment]
Do anybody have a example for an working input. Because i have some Problems there how the file needs to Look. Like

**Comment by druckgott** [comment]
You can also read the log files here: https://blackbox.betaflight.com/ I find out ;-). This should solve your problem.

**Comment by Tonynux** [comment]
> Do anybody have a example for an working input. Because i have some Problems there how the file needs to Look. Like

You can ask to Mark Spatz for a working input, I'm sure He has got has one.

**Comment by Tonynux** [comment]
> You can also read the log files here: https://blackbox.betaflight.com/ I find out ;-). This should solve your problem. It also reads Inav log files. I did not know it. But its maybe a solution.

I appreciate your willingness to help INAV Users, but we have no problems reading Logs. INAV Blackbox Explorer is OK. We only have problems with Export/Import of workpaces, and... no, unfortunately Betaflight Blackbox Explorer does not solve our problem.

**Comment by druckgott** [comment]
Yes i know, but for workspace it is maybe a help of anybody needs this. I did Not know that it works ro read the files there also.
